packages/fx-core/src/question/options/AddAuthActionOptions.ts

```diff
@@ -38,8 +38,42 @@ export const AddAuthActionOptions: CLICommandOption[] = [
     type: "string",
     description: "The authentication type for the API.",
     required: true,
-    default: "none",
-    choices: ["none", "api-key", "bearer-token", "microsoft-entra", "oauth"],
+    default: "bearer-token",
+    choices: ["bearer-token", "api-key", "oauth"],
```

Comment:
Missing microsoft entra?

Reply:
AAD is not supported for this PR. Will add later.